Examples/test-suite/python/keyword_rename_runme.py

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env python
 import keyword_rename
 keyword_rename._in(1)
+keyword_rename._in(_except=1)
```

Comment:
`%feature("kwargs");` in keyword_rename.i should fix this for -fastproxy/-builtin builds.